### PR TITLE
Moved Standard effects mapping documentation into manual

### PR DESCRIPTION
**Pull request by deborahtrez**
With reference to this [issue](https://github.com/mixxxdj/manual/issues/289), I have moved the standard effects mapping documentation to section 11.5. 
@ronso0 @Be-ing @Holzhaus  ready for your review :)
@ywwg

### CODE REVIEW

File: source/chapters/effects.rst [hunk 1]
```diff
@@ -161,3 +161,24 @@ audience, but this requires a few more steps:
...
   #. Disable the headphone button for the effect unit so you do not forget to
      turn it off later
+
+Controller Effects Mapping
+==========================
+This section describes how to use the standard Mixxx mapping for effects sections on controllers with the typical layout of 4 knobs (or 3 knobs + 1 encoder) and 4 buttons for effects. It allows you to switch between controlling all 3 effects of a chain or controlling each parameter of one effect.
+
+By default, 3 knobs are used for controlling effect metaknobs and the buttons under them toggle each effect on/off. To temporarily toggle an effect on/off, press and hold an effect enable button. The 4th knob is used for the mix (dry/wet) knob of the whole chain. On controllers designed for Serato, the :guilabel:`Beats` encoder is used as the mix knob.
+
+The button next to the mix knob is used for focusing effects. On controllers designed for Serato, this button is labeled :guilabel:`Tap`. While holding this button pressed, the enable buttons for each effect switch to choosing which effect is focused. When an effect is focused;
```
Comment (ronso0) [inline source/chapters/effects.rst:171]:
```suggestion
The button next to the mix knob is used for focusing effects. On controllers designed for Serato, this button is labeled :guilabel:`Tap`. While holding this button pressed, the enable buttons for each effect switch to choosing which effect is focused. When an effect is focused:
```
just amend that change (and other potential small fixes) to the commit and change the commit message "end of file marker?" > "document standart effects mapping"

**Review by ronso0 — CHANGES_REQUESTED** [review]
LGTM, there's only a typo that should be fixed

File: source/chapters/effects.rst (hunk 1 above)
Comment (deborahtrez) [reply source/chapters/effects.rst:171]: Alright. Let me do that right now.

**Comment by ronso0** [comment]
Thank you!

For future PRs where the change set is rather minimal there are various tools to keep the commit history clean.
Please look into `git rebase -i ...` as well as `git --fixup ...` used with `git rebase -i --autosquash`
A good explanation is here https://fle.github.io/git-tip-keep-your-branch-clean-with-fixup-and-autosquash.html

**Comment by deborahtrez** [comment]
> Thank you!
> 
> For future PRs where the change set is rather minimal there are various tools to keep the commit history clean.
> Please look into `git rebase -i ...` as well as `git --fixup ...` used with `git rebase -i --autosquash`
> A good explanation is here https://fle.github.io/git-tip-keep-your-branch-clean-with-fixup-and-autosquash.html

Thank you Ronso0. I will definitely do this next time.

**Pull request merged**